components/lcd/esp_lcd_ili9881c/esp_lcd_ili9881c.c

```diff
@@ -1,267 +1,75 @@
 /*
- * SPDX-FileCopyrightText: 2023-2024 Espressif Systems (Shanghai) CO LTD
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <stdlib.h>
-#include <sys/cdefs.h>
-#include "freertos/FreeRTOS.h"
-#include "freertos/task.h"
+#include "soc/soc_caps.h"
+
+#if SOC_MIPI_DSI_SUPPORTED
```

Comment:
if we want to use a macro to control the build of a source file, we can also use the CMake file, this macro is also available in the CMake.

Reply:
Got it. However, considering that the Arduino LCD Panel library will also migrate this component in the future, I think it's more appropriate to use macro checks in the C file.